examples/transaction-0.json

```diff
@@ -0,0 +1,302 @@
+              "code": "72166-2",
+              "system": "http://loinc.org"
+            }
+          ]
+        },
+        "effectiveDateTime": "2005-06-17",
+        "id": "0-tobacco-use",
+        "meta": {
+          "profile": [
+            "https://fhir.bbmri.de/StructureDefinition/TobaccoUse"
+          ]
+        },
+        "resourceType": "Observation",
+        "status": "final",
+        "subject": {
+          "reference": "Patient/0"
+        },
+        "valueCodeableConcept": {
+          "coding": [
+            {
+              "code": "LA18977-1",
+              "system": "http://loinc.org"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "fullUrl": "1d491668-cb18-4c5b-be24-8217f8d3380b",
+      "request": {
+        "method": "PUT",
+        "url": "Specimen/0-specimen"
+      },
+      "resource": {
+        "collection": {
+          "bodySite": {
+            "coding": [
+              {
+                "code": "C26.1",
+                "system": "https://www.who.int/classifications/icd/adaptations/oncology/en/"
+              }
+            ]
+          },
+          "collectedDateTime": "2005-06-17",
+          "fastingStatusCodeableConcept": {
+            "coding": [
+              {
+                "code": "NF",
+                "system": "http://terminology.hl7.org/CodeSystem/v2-0916"
+              }
+            ]
+          }
+        },
+        "extension": [
+          {
+            "url": "https://fhir.bbmri.de/StructureDefinition/StorageTemperature",
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "code": "temperature2to10",
+                  "system": "https://fhir.bbmri.de/CodeSystem/StorageTemperature"
+                }
+              ]
+            }
+          },
+          {
+            "url": "https://fhir.bbmri.de/StructureDefinition/SampleDiagnosis",
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "code": "B59.8",
+                  "system": "http://hl7.org/fhir/sid/icd-10"
+                }
+              ]
+            }
+          },
+          {
+            "url": "https://fhir.bbmri.de/StructureDefinition/Custodian",
+            "valueReference": {
+              "reference": "Organization/collection-2"
+            }
+          }
+        ],
+        "id": "0-specimen",
+        "meta": {
+          "profile": [
+            "https://fhir.bbmri.de/StructureDefinition/Specimen"
+          ]
+        },
+        "resourceType": "Specimen",
+        "subject": {
+          "reference": "Patient/0"
+        },
+        "type": {
+          "coding": [
+            {
+              "code": "whole-blood",
+              "system": "https://fhir.bbmri.de/CodeSystem/SampleMaterialType"
+            }
+          ]
+        }
+      }
+    }
+  ],
+  "id": "6c4f4112-5f04-4ce0-a843-540698e891dd",
+  "resourceType": "Bundle",
+  "type": "transaction"
+}
```

Comment:
Missing newline

Reply:
Ah shit. You are right. :-) Have to add it to the generator.

Reply:
Fixed